curriculum/challenges/english/00-certifications/scientific-computing-with-python-certification/scientific-computing-with-python-certification.yml

```diff
@@ -1,15 +1,15 @@
-id: 5e44431b903586ffb414c951
+challengeId: 5e44431b903586ffb414c951
```

Comment:
Why are some of these changed from `id` to `challengeId`, and not others?

Reply:
Funny story.  Crowdin regenerates our Yaml and so the formatting changes between English and the rest. Which I didn't know until just now...

Thanks, I'll fix it!